box-php/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM container-gcloud.smals.be/base-php7-fpm:3.8
```

Comment:
This is not accessible from external. This should not exist. Remove this pull request and branch.